rubric.md

```diff
@@ -0,0 +1,25 @@
+# Rubric
```

Comment:
Would you mind commenting on whether or not anything changed from the previous rubric we used?